service.subtitles.legendasdivx/changelog.txt

```diff
@@ -0,0 +1,82 @@
+1.0.4
```

Comment:
please remove the changelog from the submission as it is no longer parsed by kodi since kodi krypton. Use the `<news>` element in addon.xml to inform users about the news on the last version: https://kodi.wiki/view/Addon.xml#.3Cnews.3E

Reply:
ok